uportal-war/src/main/java/org/apereo/portal/security/provider/PersonImpl.java

```diff
@@ -244,13 +242,13 @@ public void setFullName(String sFullName) {
      */
     @Override
     public boolean isGuest() {
-        boolean isGuest = false;  // default
+        boolean rslt = false;  // default
         String userName = (String) getAttribute(IPerson.USERNAME);
-        if (PersonFactory.GUEST_USERNAME.equalsIgnoreCase(userName) && 
+        if (PersonFactory.GUEST_USERNAMES.contains(userName) &&
```

Comment:
The method could be implemented exactly like that -- and it would be correct, and it would be fine -- but here I'm modestly concerned about readability by humans.  There are a lot of things happening on that 2nd line in your example... it might be too easy to twist the logic trying to hold it in one's brain all at once.  Some times it's more readable when it's broken down into steps.

For my part, I mostly followed the number of steps that was already there.  I didn't bring much fresh energy to thinking about the preferable number of steps.

I think there are some who would have looked at those lines and broke it down more.